src/App.js

```diff
@@ -25,11 +26,12 @@ class App extends Component {
       <Router>
       <MuiThemeProvider>
         <div className="App">
-          <Route exact path="/login" component={LoginPage} />
-          <Route exact path="/signup" component={SignupPage} />
-          <Route exact path="/InternalContactOverview" component={InternalContactOverview} />
-          <Route exact path="/forgotpassword" component={ ForgotPassword }/>
-          <Route exact path="/flexicon/users/:usersId" component={ContactPage}/>
+          <Route exact path="/login" component={ LoginPage } />
+          <Route exact path="/signup" component={ SignupPage } />
+          <Route exact path="/create/order" component={ OrderCreator } />
+          <Route exact path="/create/InternalContactOverview" component={ InternalContactOverview } />
```

Comment:
and the two create/ routes should be flexicon/create/...

Reply:
Why should we have some with flexicon/ and some not?

Reply:
because the navbar is a component that will be on all the pages with /flexicon

Reply:
oooh cool! :smile: